generator/templates/schemavalidator.gotmpl

```diff
@@ -158,7 +158,7 @@ if err := validate.Required({{ if .Path }}{{ .Path }}{{else}}""{{end}}, {{ print
 {{ if .IsNullable }}if {{ .ValueExpression }} != nil {
 {{ end }}
 if err := {{.ValueExpression}}.Validate(formats); err != nil {
-  return err
+  return err.(*errors.Validation).ValidateName({{ if .Path }}{{ .Path }}{{else}}""{{end}})
```

Comment:
we may want to make this a safer cast.

```go
if ve, ok := err.(*errors.Validation); ok {
  ve.ValidateName(...
}
```

I think it will otherwise panic if it turns out to be something other than a *errors.Validation